### ISSUE TITLE
Certain exercises not working

### ISSUE THREAD

**Issue by LukusGajus30** [smintf/autolingo#27]
Matching pairs

https://user-images.githubusercontent.com/126022857/220459174-e101d4be-e016-4e78-964f-d16bd8dbd2a4.mov

Fill in the blanks (with images)

https://user-images.githubusercontent.com/126022857/220460221-15d904d0-72bd-42f2-b99c-396be91f0bf0.mov

not being impatient but is this gonna get updated eventually?

**Comment by nikitos123456789** [comment]
Yeah blanks not working for me either, it does work on less developed languages like Zulu and others which don’t have the fill in the blanks. It doesn’t work in Spanish Russian or french for me. Which language are you using it on?

**Comment by LukusGajus30** [comment]
The matching pairs video is from the Esperanto course (pretty much for every course). The fill in the blanks was recorded from the French course (the ones with images are for the major courses, like Spanish, French)

**Comment by LukusGajus30** [comment]
grammar concepts (german and spanish)

https://user-images.githubusercontent.com/126022857/220665269-ddbc9173-8721-4ef1-96ca-364b949c424f.mov

https://user-images.githubusercontent.com/126022857/220667387-d4581ee6-a302-40b6-8c2d-1da22aec784e.mov

**Comment by nikitos123456789** [comment]
yeah thats exactly the issue im having :/
hopefully the issue is updated.

**Comment by nikitos123456789** [comment]
YES, I AM GETTING TONS OF ERRORS in the extension developer part of my browser. The common one being about a 'typing enabled' part of the code? had issue before and is persistent on and off???

reply thanks :)

**Comment by nikitos123456789** [comment]
i got it working, look at my recent issues post or my repository of autolingo titled FIXED. 👍
